src/zebras/__init__.py

```diff
@@ -31,3 +31,9 @@
 from ._split_frame import split_frame, SplitFrame
 from ._timeframe import time_frame, TimeFrame
 from ._time_series import time_series, TimeSeries
+
+
+def batch(xs: list):
+    assert xs
+
+    return xs[0]._batch(xs)
```

Comment:
This looks a bit odd. Why don't we make `_batch` a class/static method?

Reply:
This is currently a hacky way to have a `batch` function. `_batch` is in fact a `classmethod`.

Reply:
(Actually, that's what it should be, but it isn't done consistently).

Alternatively we could just have functions for each type.